validator/specs/lib/validateTilesetSpec.js

```diff
@@ -77,3 +78,98 @@ describe('validateTileset', function() {
             }), done).toResolve();
     });
 });
+
+
+describe('validateBoundingVolume', function() {
+   var regionTileset = {
+       "root": {
+           "boundingVolume": {
+               "region": [10, 10, 10, 10, 10, 88]
+           },
+           "content": {
+               "boundingVolume": {
+                   "region":  [0, 0, 0, 0, 20, 30]
```

Comment:
My understanding of this might be slightly incorrect, but I updated it to be a tileset that I thought should work, namely:
`content: [18, 14, 15, 8]` and `tile: [20, 40, 50, 10]`